### ISSUE TITLE
508-defect-2 [COGNITION]: STEM Standalone - Social Security Number Alert should say "Social Security Number" instead of "SSN"

### ISSUE THREAD

**Issue by noahgelman** [department-of-veterans-affairs/va.gov-team#18159]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

Noah Gelman

## User Story or Problem Statement

As a screen reader user, a user with a cognition issue, the error text not matching the form field text can be confusing.

## Details

Social Security Number Alert should announce as "Social Security Number" instead of "SSN"

## Acceptance Criteria

- [ ] SSN alert text should be wrapped in a span tag with an aria-label: `<span aria-label="social security number">SSN</span>` OR replace the SSN text with Social Security Number

## Steps to Recreate

https://staging.va.gov/education/other-va-education-benefits/stem-scholarship/apply-for-scholarship-form-22-10203/applicant/information

Click on Social Security Field, then click off of Social Security Field

## WCAG or Vendor Guidance (optional)

https://www.w3.org/TR/UNDERSTANDING-WCAG20/minimize-error-identified.html

**Comment by ksantiagomorgan6** [comment]
PR: https://github.com/department-of-veterans-affairs/vets-website/pull/15654

**Comment by delli-gatti-michael** [comment]
@noahgelman  This is on staging and ready for review

**Comment by noahgelman** [comment]
@delli-gatti-michael Approved
